application/src/main/kotlin/net/corda/v5/application/messaging/FlowSession.kt

```diff
@@ -127,10 +122,6 @@ interface FlowSession {
  * Serializes and queues the given [payload] object for sending to the counterparty. Suspends until a response is
  * received, which must be of the given [R] type.
  *
- * Remember that when receiving data from other parties the data should not be trusted until it's been thoroughly
- * verified for consistency and that all expectations are satisfied, as a malicious peer may send you subtly corrupted
- * data in order to exploit your code.
- *
  * Note that this function is not just a simple send+receive pair: it is more efficient and more correct to use this
```

Comment:
Should this be updated, as above?

Reply:
yes the untrusted data verification has been removed from this API